packages/mobile/src/components/host-remix-contest-drawer/HostRemixContestDrawer.tsx

```diff
@@ -11,33 +11,51 @@ import {
 import { useHostRemixContestModal } from '@audius/common/store'
 import { EventEntityTypeEnum, EventEventTypeEnum } from '@audius/sdk'
 import dayjs from 'dayjs'
+import { Platform, ScrollView } from 'react-native'
 
 import {
   Flex,
   Text,
   IconTrophy,
-  Hint,
   IconCalendarMonth,
-  Button
+  Button,
+  TextLink
 } from '@audius/harmony-native'
+import { makeStyles } from 'app/styles'
 
-import { DateTimeInput } from '../core'
+import { DateTimeInput, TextInput } from '../core'
 import { AppDrawer } from '../drawer'
 
 const MODAL_NAME = 'HostRemixContest'
 
+const maxDescriptionLength = 1000
```

Comment:
should probably move these values to common somewhere. see [here](https://github.com/AudiusProject/audius-protocol/blob/7b01e4b4bab01ccf39de656bf3334772df86449f/packages/sdk/src/sdk/api/tracks/constants.ts#L1)